src/wmnds/assets/sass/_utilities.scss

```
@@ -21,6 +21,25 @@
   }
 }
 
+//Group buttons 
+.wmnds-group-buttons {
+
+  & > *:not(:first-child){
```

Whitespace sass-lint error
```suggestion
  & > *:not(:first-child) {
```